.gitpod.yml

```diff
@@ -0,0 +1,24 @@
+tasks:
+  - init: |
+      mvn -am -pl war,bom -P quick-build install
```

Comment:
Could add `-ntp` here but I guess it is valuable to see what is taking so long.

Reply:
Yeah have thought about it but I think it’s better to see in this case